icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc

```diff
@@ -257,7 +285,7 @@ vector<pair<sbn::crt::CRTHit, vector<int>>> CRTHitRecoAlg::CreateCRTHits(vector<
 //--------------------------------------------------------------------------------------------
 // Function to make filling a CRTHit a bit faster
 sbn::crt::CRTHit CRTHitRecoAlg::FillCRTHit(vector<uint8_t> tfeb_id, map<uint8_t,vector<pair<int,float>>> tpesmap,
-                            float peshit, uint64_t time0, uint64_t time1, int plane, 
+                            float peshit, uint64_t time0, Long64_t time1, int plane, 
```

Comment:
I did not change the sbnobj/comm/crthit yet. But you are right! This should be done asap.
Anyway, we shouldn't have problem casting a Long64_t for a Uint64_t.
e.g with the biggest number.
`ULong64_t c=9223372036854775807
 Long64_t e=(Long64_t) c
(long long) 9223372036854775807`
The problem is doing the viceversa.
But I totally agree with you, this should be changed in sbnobj/common/CRTHit, but I was waiting to have a general discussion at the CRT WG because I think there are proposal for other changes too.